internal/services/compute/orchestrated_virtual_machine_scale_set.go

```diff
@@ -135,73 +135,73 @@ func OrchestratedVirtualMachineScaleSetLinuxConfigurationSchema() *pluginsdk.Sch
-// 		Type:     pluginsdk.TypeSet,
-// 		Optional: true,
-// 		Computed: true,
-// 		Elem: &pluginsdk.Resource{
-// 			Schema: map[string]*pluginsdk.Schema{
-// 				"name": {
-// 					Type:         pluginsdk.TypeString,
-// 					Required:     true,
-// 					ValidateFunc: validation.StringIsNotEmpty,
-// 				},
-
-// 				"publisher": {
-// 					Type:         pluginsdk.TypeString,
-// 					Required:     true,
-// 					ValidateFunc: validation.StringIsNotEmpty,
-// 				},
-
-// 				"type": {
-// 					Type:         pluginsdk.TypeString,
-// 					Required:     true,
-// 					ValidateFunc: validation.StringIsNotEmpty,
-// 				},
-
-// 				"type_handler_version": {
-// 					Type:         pluginsdk.TypeString,
-// 					Required:     true,
-// 					ValidateFunc: validation.StringIsNotEmpty,
-// 				},
-
-// 				"auto_upgrade_minor_version": {
-// 					Type:     pluginsdk.TypeBool,
-// 					Optional: true,
-// 					Default:  true,
-// 				},
-
-// 				"force_update_tag": {
-// 					Type:     pluginsdk.TypeString,
-// 					Optional: true,
-// 				},
-
-// 				"protected_settings": {
-// 					Type:         pluginsdk.TypeString,
-// 					Optional:     true,
-// 					Sensitive:    true,
-// 					ValidateFunc: validation.StringIsJSON,
-// 				},
-
-// 				"provision_after_extensions": {
-// 					Type:     pluginsdk.TypeList,
-// 					Optional: true,
-// 					Elem: &pluginsdk.Schema{
-// 						Type: pluginsdk.TypeString,
-// 					},
-// 				},
-
-// 				"settings": {
-// 					Type:             pluginsdk.TypeString,
-// 					Optional:         true,
-// 					ValidateFunc:     validation.StringIsJSON,
-// 					DiffSuppressFunc: pluginsdk.SuppressJsonDiff,
-// 				},
-// 			},
-// 		},
-// 	}
-// }
+func OrchestratedVirtualMachineScaleSetExtensionsSchema() *pluginsdk.Schema {
+	return &pluginsdk.Schema{
+		Type:     pluginsdk.TypeSet,
+		Optional: true,
+		Computed: true,
+		Elem: &pluginsdk.Resource{
+			Schema: map[string]*pluginsdk.Schema{
+				"name": {
+					Type:         pluginsdk.TypeString,
+					Required:     true,
+					ValidateFunc: validation.StringIsNotEmpty,
+				},
+
+				"publisher": {
+					Type:         pluginsdk.TypeString,
+					Required:     true,
+					ValidateFunc: validation.StringIsNotEmpty,
+				},
+
+				"type": {
+					Type:         pluginsdk.TypeString,
+					Required:     true,
+					ValidateFunc: validation.StringIsNotEmpty,
+				},
+
+				"type_handler_version": {
+					Type:         pluginsdk.TypeString,
+					Required:     true,
+					ValidateFunc: validation.StringIsNotEmpty,
+				},
+
+				"auto_upgrade_minor_version": {
+					Type:     pluginsdk.TypeBool,
+					Optional: true,
+					Default:  true,
+				},
+
+				"force_update_tag": {
+					Type:     pluginsdk.TypeString,
+					Optional: true,
+				},
+
+				"protected_settings": {
+					Type:         pluginsdk.TypeString,
+					Optional:     true,
+					Sensitive:    true,
+					ValidateFunc: validation.StringIsJSON,
+				},
+
+				"provision_after_extensions": {
```

Comment:
provision what? should this be provision_this_after_extensions

Reply:
Fixed... I updated it to be `extensions_to_provision_after_vm_creation` which is exactly what this field does.